frontend/src/components/explorer/mapViewer/MapsListing.vue

```diff
@@ -2,31 +2,33 @@
   <div id="maps-listing">
     <div v-for="category in Object.keys(mapsListing).filter(c => mapsListing[c].length > 0).sort()"
          :key="category" class="card my-3">
-      <p class="is-capitalized is-size-6 has-text-weight-bold">{{ category.replace(/.$/," maps") }}</p>
-      <span v-for="item in mapsListing[category]" :key="item.id">
-        <template v-if="showing2D">
-          <template v-if="item.svgs.length === 0">
-            {{ item.name }}
-          </template>
-          <template v-else-if="item.svgs.length === 1">
-            <a @click="changeToMap(item.svgs[0].id)">
-              {{ item.svgs[0].customName }}
-            </a>
+      <template v-if="category !== 'customs' || showing2D">
```

Comment:
Since this `v-if` is inside of the `v-for`, the `div` from line 3 is still being rendered, which is causing the visual bug @mihai-sysbio mentioned [here](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/643#pullrequestreview-694091438).

A suggestion is to combine the logic from `v-for` and `v-if` into a computed property, perhaps called `categories`.

Reply:
Thanks a lot @mihai-sysbio and @e0 for pointing out this problem. I will try to solve it with Shan's suggestion.

Reply:
Hi @mihai-sysbio and @e0, I've added a computed `categories` to solve the problem mentioned above.